### PR TITLE
Improving the documentation of learning_rate_scheduler.py

### PR DESCRIPTION
**Pull request by vsuryamurthy**
I have removed the 'global_step' from the usage example of various learning rate scheduler as it is no longer in use.
 
This is in reference to https://github.com/tensorflow/tensorflow/issues/30437

### CODE REVIEW

**Comment by vsuryamurthy** [comment]
I signed it!

**Comment by vsuryamurthy** [comment]
I signed it!

**Comment by vsuryamurthy** [comment]
I signed it!

**Comment by vsuryamurthy** [comment]
I signed it!

**Comment by vsuryamurthy** [comment]
signed

**Comment by vsuryamurthy** [comment]
Any change

**Comment by vsuryamurthy** [comment]
hello

**Comment by vsuryamurthy** [comment]
I signed

**Comment by vsuryamurthy** [comment]
success

**Comment by vsuryamurthy** [comment]
signed

**Review by tomerk — APPROVED** [review]
